RepoDb.Core/RepoDb/Extensions/DbCommandExtension.cs

```diff
@@ -268,6 +268,12 @@ private static void EnsureTableValueParameter(IDbDataParameter parameter)
                     dbType = Converter.EnumDefaultDatabaseType;
                 }
 
+                // DbType fallback
+                if (dbType == null)
+                {
+                    dbType = TypeMapper.GetFallback(classProperty.PropertyInfo.PropertyType);
```

Comment:
We do not need this additional mapping anymore as the code at the line [261](https://github.com/mikependon/RepoDB/blob/5472110ac13c9b816f9cc77aaa87e54cf706c4c7/RepoDb.Core/RepoDb/Extensions/DbCommandExtension.cs#L261) should do this already. If there are issues to this, we should fix it there somehow, not introducing this new mapping.

```csharp
var dbType = (returnType != null ? clientTypeToDbTypeResolver.Resolve(returnType) : null) ??
                    classProperty.GetDbType() ??
                    value?.GetType()?.GetDbType();
```